.infra/index.ts

```diff
@@ -306,10 +306,6 @@ if (isAdhocEnv) {
         { targetPort: 3000, port: 80, name: 'http' },
         { targetPort: 9464, port: 9464, name: 'metrics' },
       ],
-      spot: {
-        enabled: true,
-        weight: 50,
-      },
```

Comment:
I don't see a reason to prevent API from running on spot nodes, it's been running on them since performance sprint, and haven't caused any issues

Reply:
It shouldn't really affect it, but that's fine.
I really wish it was possible to say "Always run x of y replicas on these nodes"

Reply:
yap once it is possible we can consider. also this will give us good look if it really changes anything to put it on spot nodes vs not